app/assets/stylesheets/_forms.scss

```diff
@@ -8,7 +8,7 @@ fieldset {
 input,
 label,
 select {
-  display: block;
+  display: inline;
```

Comment:
@whmii Have you tested this change? I think changing this to `inline` could cause a lot of issues. I’m sure we don’t want all `label`’s to be `inline`.